Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -1 +1,271 @@
 
+## Notes for Flutter TDD Clean Architecture Course Videos
+
+#### 1. Explanation & Project Structure
+> Video:https://www.youtube.com/watch?v=KjE2IDphA_U&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech
+
+In the `first tutorial` an explanation is given of the importance of having all functions tested in our application.  
+It also explains the **clean architecture**.
+All the features of the application will be divided into three folders, **domain**, **data** and **presentation**.
+It is also explained how each of these folders works and what they will contain.
+
+<br />
+
+#### 2. Entities & Use Cases 
+> Video:https://www.youtube.com/watch?v=lPkWX8xFthE&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech&index=2
+
+In the `second tutorial` we start by adding the packages of the dependencies that will be needed to build the application.
+
+````
+dependencies:
+  flutter:
+    sdk: flutter
+
+  get_it: ^2.0.1
+  
+  # Bloc for state management
+  flutter_bloc: ^0.21.0
+  
+  # Value equality
+  equatable: ^0.4.0
+  
+  # Functional programming thingies
+  dartz: ^0.8.6
+  
+  # Remote API
+  connectivity: ^0.4.3+7
+  
+  http: ^0.12.0+2
+  # Local cache
+  shared_preferences: ^0.5.3+4
+
+  cupertino_icons: ^0.1.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  mockito: ^4.1.0
+  ````
+  
+  It is explained how the **entity** will work in this application.
```

Comment:
you've used the word **entity** but not explained what it is. 
If the video explains something, you need to _summarise_ it not just say, the "video explains".
Otherwise there is no point to making notes if you are just going to force the reader to go and re-watch the video to understand what an **entity** is. The point of _summarising_ the video is for you to use your _own_ words to explain the concepts so that you confirm your _own_ understanding. 💭

Reply:
I get it, I'm gonna change it.